tests/components/application-connector/resources/charts/gateway-test/templates/applications/body-forwarding.yml

```diff
@@ -0,0 +1,38 @@
+apiVersion: applicationconnector.kyma-project.io/v1alpha1
+kind: Application
+metadata:
+  name: body-forwarding
```

Comment:
I think the methods are more key in this test, aren't they? Maybe something like `methods-with-body`